src/common/websocket/config.js

```diff
@@ -6,64 +6,62 @@
  * git update-index --assume-unchanged src/javascript/config.js
  *
  */
-const domain_app_ids = {
-    // these domains as supported "production domains"
-    'deriv.com': 11780,
+const domain_config = {
```

Comment:
I think production & staging don't matter here, as users won't be able to log in. also, there will be the same redirect url.

Reply:
@ashkanx The app id is used for the websocket connection and we need the hostname to match `app_id` with environment